packages/client/hmi-client/src/components/dataset/tera-dataset.vue

```diff
@@ -386,7 +388,7 @@ async function updateDatasetName() {
 		datasetClone.name = newDatasetName.value;
 		await updateDataset(datasetClone);
 		dataset.value = await getDataset(props.assetId);
-		useResourcesStore().setActiveProject(await ProjectService.get(props.project.id, true));
+		getActiveProject(props.project.id);
```

Comment:
Do we need to do that? Should we just update our active project assets lost with the new name of the Dataset once we got the response that the name has changed? @mwdchang what do you think, for things like names, does it matter?

Reply:
I'm saying, let's not make another call to refresh the whole project, and just update the local object. Which does the same job on the front-end, but only to what matter.

Reply:
I think this has to do with the shallow-ref vs regular-ref discussion here https://github.com/DARPA-ASKEM/Terarium/pull/1785#discussion_r1324662310

I don't have a strong feeling towards one or the other, as long as the mechanism is well understood by all.

Reply:
@YohannParis We shouldn't update the local object because then it may get out of sync - precisely what we're trying to fix. In fact, there doesn't even need to be a local object, and I should just delete the `project` prop. `activeProject` needs to be refreshed so that any changes are propagated to all its users.